src/cart/crt_hg.c

```diff
@@ -327,6 +467,88 @@ crt_hg_addr_lookup(struct crt_hg_context *hg_ctx, const char *name,
 	return rc;
 }
 
+int
+crt_hg_remove_addr(hg_class_t *hg_class, hg_addr_t hg_addr)
+{
+	hg_return_t     ret = HG_SUCCESS;
+
+	D_DEBUG(DB_TRACE, "removing hg addr %" PRIu64 "\n",
+			(uint64_t)hg_addr);
+	D_DEBUG(DB_TRACE, "removing  hg class %" PRIu64 "\n",
+			(uint64_t)hg_class);
```

Comment:
better to combine these 2 into single log line. on busy systems this can end up being printed far apart with multiple threads logging stuff, so we might lose track of what things go together